docker/all-in-one/init/configure-admin-mgr.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eou pipefail
+
+ADMINMGR_CUSTOM_DIR="${DATA_VOLUME_MOUNTPOINT}/admin-mgr"
+
+if [ "${DATA_VOLUME_MOUNTPOINT}" ]; then
+  mkdir -p "${ADMINMGR_CUSTOM_DIR}"
+
+  if [ ! -f "${ADMINMGR_CUSTOM_DIR}/admin-mgr" ]; then
+    echo "Copying existing admin-mgr binary from /usr/bin/admin-mgr to ${ADMINMGR_CUSTOM_DIR}"
+    cp "/usr/bin/admin-mgr" "${ADMINMGR_CUSTOM_DIR}/admin-mgr"
```

Comment:
Not sure what the ideal place is since we're doing this for each of the services, but the general flow of operations here could use a comment describing it

Reply:
Though about pulling this out in a generic function - but its maintainability might take a hit given the number of args and options per service. Will take a different approach, so that we also save on boot time overhead